static/app.template.html

```diff
@@ -1,5 +1,6 @@
 <html>
 	<head>
+		<title>ISummary</title>
```

Comment:
Isn't better maintain the app name, don't yout think? Or something like "Isumm - Summary"